js/ui/drop_down_button.js

```diff
@@ -635,6 +641,22 @@ let DropDownButton = Widget.inherit({
         this._popup && this._popup.repaint();
     },
 
+    _selectModeChanged(value) {
+        if(this._list) {
+            this._setListOption('selectionMode', value ? 'single' : 'none');
+            if(value) {
```

Comment:
What about value = 0 case?

Reply:
It is the useSelectMode option value and has a type: "boolean".

Reply:
Got it.